src/features/NewPetitionFromRecordSpreadsheet.jsx

```diff
@@ -0,0 +1,183 @@
+import React, { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import { motion, AnimatePresence } from 'framer-motion';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import { Button, CloseButton } from '../components/elements/Button';
+import DragNDrop from '../components/elements/DragNDrop/DragNDrop';
+import { useCreateBatchFromRecordSpreadsheetMutation } from '../service/api';
+import { Spinner } from '../components/elements/Spinner';
+import { POSITIVE } from '../components/elements/Button/Button';
+
+const ALLOWED_MIME_TYPES = [
+  'application/vnd.ms-excel',
+  'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+];
+const MAX_FILES = 1;
+const MAX_FILE_SIZE = 30000;
```

Comment:
Do we really want this max file size? I can't tell if it's in bytes, KB or MB, so it may be pretty small.

Reply:
I got rid of it.